dwave/optimization/model.pyx

```diff
@@ -1764,7 +1764,15 @@ cdef class ArraySymbol(Symbol):
         if exponent == 2:
             from dwave.optimization.symbols import Square  # avoid circular import
             return Square(self)
-        raise ValueError("only squaring is currently supported")
+        # check if exponent is an integer greater than 0
+        elif isinstance(exponent, numbers.Real) and exponent > 0 and int(exponent) == exponent:
+            expanded = itertools.repeat(self, int(exponent))
+            out = next(expanded)  # get the first one
+            # multiply self by itself exponent times
+            for symbol in expanded:
+                out = out.__imul__(symbol)
```

Comment:
Why not just use `*=`? It's the same thing and a bit more readable. Also more immune to later changes we might make.

Reply:
I will change it back. I changed from `*=` to `__imul__` because I was wondering if that was the cause of failing the compatibility tests (which it was not).